src/dialogs/voice-command-dialog/ha-voice-command-dialog.ts

```diff
@@ -13,16 +12,24 @@ import {
   customElement,
   query,
   PropertyValues,
+  TemplateResult,
 } from "lit-element";
 import { HomeAssistant } from "../../types";
 import { fireEvent } from "../../common/dom/fire_event";
 import { processText } from "../../data/conversation";
 import { classMap } from "lit-html/directives/class-map";
 import { PaperInputElement } from "@polymer/paper-input/paper-input";
+import { haStyleDialog } from "../../resources/styles";
+// tslint:disable-next-line
+import { PaperDialogScrollableElement } from "@polymer/paper-dialog-scrollable/paper-dialog-scrollable";
+
+type MessageType = "text" | "link" | "image" | "choice";
 
 interface Message {
   who: string;
-  text: string;
+  text?: string;
+  type?: MessageType;
+  data?: any;
```

Comment:
We shouldn't `any` this. We should have 4 interfaces, one `TextMessage` with `type: "text"` etc. and then `message = TextMessage | LinkMessage`